_data/students.yml

```diff
@@ -587,4 +587,9 @@
   facebook: Baidya99
   image: Baidya99.jpg
   bio: Loves tech and also coding.
+  
+- name: Zhang Jing Yuan
+  github: veehshiftz
+  image: table.jpeg
```

Comment:
@nikunjness Please check gh pages once before merging. 
@veehshiftz You have written table.jpeg here and the image is table.jpg. Please create a new PR with the updated image. Thanks!